### ISSUE TITLE
Theme problems 

### ISSUE THREAD

**Issue by havefun77612** [codemybrainsout/smart-app-rate#47]
I have this error here 
You need to use a Theme.AppCompat theme (or descendant) with this activity

**style.xml** 
```
<resources>

    <!-- Base application theme. -->

    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
        <!-- Customize your theme here. -->
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
    </style>


</resources>
```

**AndroidManifest.xml**

```
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    package="com.JunkyCoder.pro">

    <uses-permission android:name="android.permission.INTERNET" />
    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>

    <application
        android:allowBackup="true"
        android:icon="@mipmap/launture_round"
        android:label="@string/app_name"
        android:roundIcon="@mipmap/launture_round"
        android:supportsRtl="true"
        android:theme="@style/AppTheme"

        tools:ignore="GoogleAppIndexingWarning">
        <meta-data
            android:name="com.google.android.gms.ads.APPLICATION_ID"
            android:value="ca-app-pub-9275674846813033~2920278162"/>
        <activity android:name=".AddNewContent.AddNew"/>
        <activity android:name=".MainPage">
            <intent-filter>
                <action android:name="android.intent.action.MAIN" />

                <category android:name="android.intent.category.LAUNCHER" />
            </intent-filter>
        </activity>
    <receiver android:name=".NotificationBar.NotificationActionService"/>
        <service android:name=".NotificationBar.OnClearFromRecentService"/>
    </application>

</manifest>
```

**calling method** 

```
    public void showReviewDialog(){
        final FirstUse userRateUs =new FirstUse(mContext);
        Drawable icon=appCompatActivity.getResources().getDrawable(R.mipmap.launture);
        if (!userRateUs.getUserRateUs()){
            final RatingDialog ratingDialog = new RatingDialog.Builder(mContext)
                    .icon(icon)
                    .threshold(3)
                    .title("الي اي مدي ترشحنا ؟")
                    .titleTextColor(R.color.black)
                    .positiveButtonText("ليس الان")
                    .negativeButtonText("لا اريد")
                    .positiveButtonTextColor(R.color.white)
                    .negativeButtonTextColor(R.color.grey_500)
                    .formTitle("اوك")
                    .formHint("من فضلك اخبرنا كيف نحسن من تطبيقنا ؟ ")
                    .formSubmitText("ارسال")
                    .formCancelText("الغاء")
                    .ratingBarColor(R.color.colorPrimary)
                    .playstoreUrl("https://play.google.com/store/apps/details?id=com.JunkyCoder.pro")
                    .onThresholdCleared(new RatingDialog.Builder.RatingThresholdClearedListener() {
                        @Override
                        public void onThresholdCleared(RatingDialog ratingDialog, float rating, boolean thresholdCleared) {
                            //do something
                            ratingDialog.dismiss();
                            appCompatActivity.finish();
                        }
                    })
                    .onThresholdFailed(new RatingDialog.Builder.RatingThresholdFailedListener() {
                        @Override
                        public void onThresholdFailed(RatingDialog ratingDialog, float rating, boolean thresholdCleared) {
                            //do something
                            ratingDialog.dismiss();
                            appCompatActivity.finish();
                        }
                    })
                    .onRatingChanged(new RatingDialog.Builder.RatingDialogListener() {
                        @Override
                        public void onRatingSelected(float rating, boolean thresholdCleared) {

                        }
                    })
                    .onRatingBarFormSumbit(new RatingDialog.Builder.RatingDialogFormListener() {
                        @Override
                        public void onFormSubmitted(String feedback) {
                            sendMeEmail(feedback);
                            userRateUs.setUserRateUsToTrue();
                        }
                    })
                    .build();

            ratingDialog.show();
        }

    }

```

**Comment by pflammertsma** [comment]
It appears that smart-app-rate does not support theming, unless I have overlooked something. In particular, dark themes appear to break the usability of the dialog quite substantially:

![image](https://user-images.githubusercontent.com/286051/79125621-bef12180-7d9e-11ea-9142-06fd769ee356.png)

**Comment by hansemannn** [comment]
Hi there! This should be fixed in #48!

**Comment by iormark** [comment]
When will appear in the release?

**Comment by ahulr** [comment]
Released [v2.0.0](https://github.com/codemybrainsout/smart-app-rate/releases/tag/2.0.0)
